sandbox-backend/src/test/java/com/exadel/sandbox/team5/service/ValidatePromoCodeGeneratorTest.java

```diff
@@ -0,0 +1,28 @@
+package com.exadel.sandbox.team5.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class ValidatePromoCodeGeneratorTest {
```

Comment:
We can also use `@RunWith(MockitoJUnitRunner.class)`

Reply:
Done.